.gitignore

```diff
@@ -1,3 +1,6 @@
+/.idea
+aws-sdk-php-silex.iml
```

Comment:
It's where IntelliJ stores metadata. Not sure if the `.iml` extension is used by anything else.